### DISCUSSION TITLE
Install cert and reload nginx without root?

### DISCUSSION THREAD

**Discussion by ajaskiewiczpl** [acmesh-official/acme.sh#4433] [Q&A]
acme.sh documentation states that root is not required for issuing a cert and that is true, however how can I use the script to automatically install and reload nginx as non-root user? So first we have to install cert for example to `/etc/nginx/ssl-cert` directory and do `service nginx force-reload`. Both commands require root/sudo. What's the recommended solution? Right now I installed acme.sh as root user on my server, however I feel like this is not right approach.

**Comment by f1-outsourcing** [comment]
I am putting a flag somewhere, if it exists a cron job reloads nginx/apache

**Comment by neilpang** [comment]
you can configure that the "reload" doens't require "sudo"
https://github.com/acmesh-official/acme.sh/wiki/sudo#visudo
